subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -664,24 +523,68 @@ static int nct_settings_init(void)
 {
 	int ret = 0;
 
-#if !IS_ENABLED(CONFIG_MQTT_CLEAN_SESSION)
+#if !IS_ENABLED(CONFIG_MQTT_CLEAN_SESSION) || IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)
```

Comment:
nit: IS_ENABLED
use define instead.

Reply:
If you want to use IS_ENABLED it should be done with 
```
if(!IS_ENABLED(CONFIG_MQTT_CLEAN_SESSION) || IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)
```

So that the following code is checked.